modules/cells/src/main/java/dmg/cells/nucleus/CellAdapter.java

```diff
@@ -75,7 +75,7 @@
 public class CellAdapter
     implements Cell, CellEventListener, CellEndpoint
 {
-    private static final Logger _log =
+    private static final Logger logger =
```

Comment:
oh, but we have been researching. Loggers are not constants according to a certain java styleguide. I'll research the link..